src/main/java/com/example/rest/UserStoryResource.java

```diff
@@ -0,0 +1,69 @@
+package com.example.rest;
+
+import com.example.authentication.Secured;
+import com.example.model.UserStory;
+import com.example.repository.UserStoryRepository;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Path("/userstories")
+public class UserStoryResource {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<UserStory> getAll() {
+        return UserStoryRepository.getAll();
+    }
+
+    @GET
+    @Path("/{userStoryId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public UserStory getUserStory(@PathParam("userStoryId") int userStoryId){
+        for(UserStory us : UserStoryRepository.getAll()){
+            if(us.getId() == userStoryId){
+                return us;
+            }
+        }
+        return null;
+    }
+
+    @POST
+    @Secured
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public String postUserStory(UserStory story){
+        UserStoryRepository.add(story);
+        return "successfully posted";
+    }
+
+    @PUT
+    @Secured
+    @Path("/{userStoryId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public String updateUserStory(@PathParam("userStoryId") int userStoryId,UserStory newStory) {
+        for (UserStory us : UserStoryRepository.getAll()) {
+            if (us.getId() == userStoryId) {
+                UserStoryRepository.remove(us);
+                UserStoryRepository.add(newStory);
+                return "Updated successfully";
+            }
+        }
+        return "Object to update not found";
```

Comment:
By convention, if an object is not found, the server should send 404, if a rest update is succesfull, it should send the updated object's json